samples/scala-value-entity-customer-registry/src/main/proto/customer/api/customer_api.proto

```diff
@@ -16,8 +16,6 @@ syntax = "proto3";
 
 package customer.api;
 
-option java_outer_classname = "CustomerApi";
```

Comment:
Why not keep them? They're ignored for Scala anyway, right?

Reply:
ok, I was uncertain if they were fully ignored or not. However, since these are to be shown in doc snippets it would be confusing to include them when they are not used.